docs/README.md

```diff
@@ -1,5 +1,7 @@
 imagetools / [Modules](modules.md)
 
+imagetools / [Modules](modules.md)
+
```

Comment:
we should manually remove this until https://github.com/tgreyuk/typedoc-plugin-markdown/issues/385 is fixed
```suggestion
```